WcaOnRails/app/views/persons/_details.html.erb

```diff
@@ -0,0 +1,34 @@
+<div>
+  <div class="text-center">
+    <h2><%= @person.name %></h2>
+    <!-- Handle multiple sub ids. -->
+    <% if @previous_persons.any? %>
+      <h4>
+        (<%= t 'persons.show.previously' %> <%= @previous_persons.map { |person| "#{person.name} - #{person.country.name}" }.join(', ') %>)
+      </h4>
+    <% end %>
+    <% if @person&.user&.avatar %>
+      <%= image_tag @person.user.avatar, class: "avatar" %>
+    <% end %>
+  </div>
+  <div class="table-responsive details">
+    <table class="table table-striped">
+      <thead>
+        <tr>
+          <th><%= t 'common.country' %></th>
+          <th><%= t 'common.user.wca_id' %></th>
+          <%= content_tag :th, t('activerecord.attributes.person.gender') if @person.gender.present? %>
```

Comment:
No need to do this as part of this PR, but see https://github.com/thewca/worldcubeassociation.org/issues/1553, which I hope you don't mind that I've assigned to you!